codegen/codegen.sh

```diff
@@ -83,6 +85,43 @@ for type in "${rekor_types[@]}"; do
         --output "${pkg_dir}/_internal/${type}.py"
 done
 
+# RekorV2
+rekorv2_ref="main" # not yet a non-prelease release.
```

Comment:
Using #main, while there is not an a "latest" release.
 - https://github.com/sigstore/rekor-tiles/releases